src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveReviewsService.kt

```diff
@@ -147,6 +154,7 @@ enum class IncentiveReviewSort(
   private val defaultOrder: Sort.Direction,
 ) {
   NEXT_REVIEW_DATE("nextReviewDate", IncentiveReview::nextReviewDate, Sort.Direction.ASC),
+  DAYS_SINCE_LAST_REVIEW("daysSinceLastReview", { it.daysSinceLastReview ?: Int.MAX_VALUE }, Sort.Direction.DESC),
```

Comment:
To allow sorting by this column, people who've never had a real review are considered to have had a review a really really long time ago.

That is, if sorting descending by "Days since last review" they will appear first, before someone who last had a review, say, 400 days ago.